### DISCUSSION TITLE
Version update etcd !!!

### DISCUSSION THREAD

**Discussion by dhamahes** [etcd-io/etcd#17226] [Q&A] [answered]
When can we expect a version update with logrus package fix?

**Comment by jmhbnz** [comment, marked as answer]
Hey @dhamahes - Thanks for raising this question. Logrus is an indirect dependency of a component called `github.com/tmc/grpc-websocket-proxy` which is in turn used by etcd server package. Resolution is tracked under https://github.com/etcd-io/etcd/issues/17268.
